src/components/AppCatalog/CatalogList/CatalogRepo.js

```diff
@@ -47,6 +55,22 @@ const Image = styled.img`
   background-color: ${({ theme }) => theme.colors.white1};
   flex-shrink: 0;
   border-radius: 5px;
+
+  &:before {
+    content: "${(props) => acronymize(props.alt)}";
```

Comment:
Can we pass the acronyms as a prop for the `Repo` component instead of having logic in the styled component?